src/exercises/day-1/book-library.rs

```diff
@@ -112,7 +112,7 @@ impl Library {
 // method signatures, including the "self" parameter! You may
 // also need to update the variable bindings within main.
 fn main() {
-    let library = Library::new();
+    let mut library = Library::new();
```

Comment:
People are supposed to run into this mistake by themselves. The exercise is there to let people work with `mut` and method receivers.

```suggestion
    let library = Library::new();
```